.github/workflows/container-images-cd.yml

```diff
@@ -13,7 +13,6 @@ on:
     push:
         branches:
             - master
-            - main
```

Comment:
This is coming from a great suggestion from @hazzadous: as we only have branch protection enabled on `master`, let's do not use `main` till we officially switch to it.